clpy/backend/__init__.py

```diff
@@ -39,6 +39,7 @@
 # from clpy.backend.stream import get_elapsed_time  # NOQA
 from clpy.backend.stream import Stream  # NOQA
 
+nvtx_enabled = False
```

Comment:
This should be implemented on `backend.profiler`, not here directly

Reply:
I will fix it

Reply:
In cupy, nvtx_enabled is defined in `cupy/cuda/__init__.py` directly.
So, I will impliment nvtx_enabled in `clpy/backend/__init__.py`.

https://github.com/cupy/cupy/blob/c97afb35da86c58d936c322c31316348454043e3/cupy/cuda/__init__.py#L24-L29

Reply:
I found there's not [the part for `*_enabled`](https://github.com/cupy/cupy/blob/v2.1.0/cupy/cuda/__init__.py#L18-L47) on ClPy. Who removed...? (might be by myself)

Reply:
@t-kitawaki Please restore that part, not implemt directly.

Reply:
@LWisteria 
CI was not passed.
Because, flake8 says `./clpy/backend/__init__.py:42:26: F821 undefined name 'runtime'` in is_available function.
May I comment out or remove is_available function?

Reply:
or # NOQA comment?

Reply:
Yes. Make it out and leave TODO comment. (get device_count() can be implemented also on OpenCL)